script/test

```diff
@@ -108,7 +108,7 @@ generate_key_and_certificate ${OTHER_TEST_PRIVATE_KEY} ${OTHER_TEST_CERTIFICATE}
 
 echo "==> Starting punica(-s)..."
 REGULAR_PUNICA_PID=$(run_punica "regular")
-SECURE_PUNICA_PID=$(run_punica "secure" "-c ${PROJECT_ROOT_DIR}/tests/rest/secure.cfg")
+SECURE_PUNICA_PID=$(run_punica "secure" "-c ${PROJECT_ROOT_DIR}/tests/rest/secure.cfg -d ${PROJECT_ROOT_DIR}/tests/rest/database.json")
```

Comment:
In https://github.com/punica/punica/pull/28 the 'regular' punica server needs a database file for tests. Since the server does not use a config file it can only be passed through command line arguments. 'secure' server also needs a database file for code coverage and I feel if we took different approaches to this it would only cause confusion when reading the test script, as one server seems to be using database file functionality while the other isn't.